eng/common/testproxy/docker-start-proxy.ps1

```diff
@@ -64,9 +64,25 @@ if ($Mode -eq "start"){
     }
     # else we need to create it
     else {
+        $attempts = 0
         Write-Host "Attempting creation of Docker host $CONTAINER_NAME"
         Write-Host "docker container create -v `"${root}:${Initial}/etc/testproxy`" $LinuxContainerArgs -p 5001:5001 -p 5000:5000 --name $CONTAINER_NAME $SelectedImage"
-        docker container create -v "${root}:${Initial}/etc/testproxy" $LinuxContainerArgs -p 5001:5001 -p 5000:5000 --name $CONTAINER_NAME $SelectedImage
+        while($attempts -lt 3){
+            docker container create -v "${root}:${Initial}/etc/testproxy" $LinuxContainerArgs -p 5001:5001 -p 5000:5000 --name $CONTAINER_NAME $SelectedImage
+
+            if($LASTEXITCODE -ne 0){
+                $attempts += 1
+                Start-Sleep -s 1
```

Comment:
Probably worth a message saying you are going to retry. Also do we think 1 second is good or should we sleep longer?

Reply:
Fixed. Thanks!